demo/demos/checkout-pricing.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Elements, RecurlyProvider, useCheckoutPricing } from '../../lib/index';
+
+export function CheckoutPricing () {
+  return (
+    <RecurlyProvider publicKey={process.env.RECURLY_PUBLIC_KEY}>
+      <Elements>
+        <CheckoutPricingForm />
+      </Elements>
+    </RecurlyProvider>
+  );
+};
+
+function CheckoutPricingForm () {
+  const [recurlyError, setRecurlyError] = useState(null);
+  const [plan, setPlan] = useState('');
+  const [quantity, setQuantity] = useState('');
+  const [pricing, setPricing] = useCheckoutPricing({ plan }, setRecurlyError);
+
+  function updatePlan (e) {
+    const plan = e.target.value;
+    setRecurlyError(null);
+    setPlan(plan);
+    setPricing({
+      plan,
+    });
+  };
+
+  function updatePricing (e) {
+    e.preventDefault();
+    setPricing({ subscriptions: [{ plan, quantity }] });
+  };
+
+  return (
+    <div className="DemoSection">
+      <div>
+        <form onSubmit={updatePricing}>
+          <div>
+            <select value={plan} onChange={updatePlan}>
+              <option value="">Select a plan</option>
+              <option value="basic">Basic</option>
+              <option value="advanced">Advanced</option>
+            </select>
+            <input
+              type="number"
+              onChange={quantity}
+              onChange={e => setQuantity(e.target.value)}
+              placeholder="Quantity"
+              min="0"
+            />
+          </div>
+          <div>
+            <button>Calculate subtotal</button>
+          </div>
+          <div style={{ marginTop: '15px' }}>
```

Comment:
I'm not sure if this is appropriate or we need to define a CSS rule for this instead.